package.json

```diff
@@ -18,34 +15,47 @@
     "lint:fix": "eslint \"src/**/**.{ts,tsx}\" --fix",
     "perf": "parcel perf/index.html",
     "build:tsc": "tsc --project ./tsconfig.json",
-    "build": "npm run clean && npm run build:tsc && parcel build",
+    "build:umd": "parcel build --target legacy --dist-dir dist/legacy",
+    "build": "npm run clean && npm run build:tsc && npm run build:umd",
+    "copy": "cp package* README.md LICENSE dist/",
     "examples:dev": "npm run clean && parcel examples/index.html",
     "docs:dev": "npm run clean && parcel docs-src/index.html",
     "docs:build": "rm -rf ./docs/ && parcel build ./docs-src/index.html --dist-dir docs/ --public-url https://sayari-analytics.github.io/trellis/ && cp ./docs-src/assets/og-image.png ./docs/"
   },
+  "targets": {
```

Comment:
I also think we can get rid of `targets`. this is a parcel-specific thing, but again, parcel only comes in to play w/ the umd build, which just targets the single entrypoint file `./src/index.umd.ts`